### PR TITLE
feat(embed): improve the way the logger embed is used

### PR DESCRIPTION
**Pull request by PedroS235**
# Description

This introduced the Enum python class to distinguish the different levels. In addition, everywhere it was used before, it was ported to the new LoggerEmbed

Fixes #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Run the bot on a template server and verified that the logger embeds are still working as expected.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/bics_bot/cogs/events/on_events.py [hunk 1]
```diff
@@ -26,6 +29,18 @@ async def on_member_join(self, member: Member):
 
         await member.send(embed=WelcomeEmbed(member.display_name, server.name))
 
+    @tasks.loop(time=datetime.time(15, 3, tzinfo=datetime.timezone.utc))
```
Comment (umutucak) [inline src/bics_bot/cogs/events/on_events.py:32]: you forgot to remove this loop from testing

File: src/bics_bot/cogs/events/on_events.py [hunk 2]
```diff
@@ -16,6 +18,7 @@ async def on_ready(self):
         is fully active.
         """
         print("Bot is fully deployed and is now online!")
+        self.check_birthday.start()
```
Comment (umutucak) [inline src/bics_bot/cogs/events/on_events.py:21]: here too

**Review by umutucak — APPROVED** [review]
i like it, it is much cleaner. clear to merge after the test task loop is removed 👍

File: src/bics_bot/cogs/events/on_events.py (hunk 2 above)
Comment (PedroS235) [reply src/bics_bot/cogs/events/on_events.py:21]: ah good catch

**Review by umutucak — COMMENTED** [review]
YAAAAAA

**Pull request merged**